### DISCUSSION TITLE
want to add a tree control to manage client-side

### DISCUSSION THREAD

**Discussion by chendx-github** [1Remote/1Remote#529] [Ideas]
**Is your enhancement related to a problem? Please describe.**

现在我的客户端非常多大概有100+，每次查找的时候很难找到
虽然标签和搜索可以解决筛选的问题，但是很多时候 记不住这些名字和标签
需要折叠之后才能快速的找到

**Describe the solution you'd like**

我想要增加一个类似mRemoteNG 这个软件的属性文件夹结构

**Describe alternatives you've considered**

**Additional context**
Add any other context or screenshots about the enhancement here.

**Comment by VShawn** [comment]
展示不会改为树状菜单，转为讨论，以备后用
